### PR TITLE
Remove obsolete methods from Akka.IO

### PR DESCRIPTION
**Pull request by Arkatufus**

### CODE REVIEW

File: src/core/Akka/IO/Inet.cs [hunk 1]
```diff
@@ -76,12 +76,6 @@ public abstract class AbstractSocketOptionV2 : SocketOptionV2 { }
         /// </summary>
         public class DatagramChannelCreator : SocketOption
         {
-            [Obsolete(message:"Please use the method with AddressFamily parameter. Using this method will always return an IPv6 bound UDP endpoint.")]
-            public virtual Socket Create()
```
Comment (Arkatufus) [inline src/core/Akka/IO/Inet.cs:80]: Obsolete since v1.4.28

File: src/core/Akka/IO/Udp.cs [hunk 2]
```diff
@@ -176,12 +176,6 @@ public NoAck(object token)
         /// </summary>
         public sealed class Send : Command
         {
-            [Obsolete("Akka.IO.Udp.Send public constructors are obsolete. Use `Send.Create` or `Send(ByteString, EndPoint, Event)` instead.")]
-            public Send(IEnumerator<ByteBuffer> payload, EndPoint target, Event ack)
```
Comment (Arkatufus) [inline src/core/Akka/IO/Udp.cs:180]: Obsolete since v1.3.13 (6 years)

File: src/core/Akka/IO/UdpConnected.cs [hunk 3]
```diff
@@ -165,12 +165,6 @@ public NoAck(object token)
         /// </summary>
         public sealed class Send : Command
         {
-            [Obsolete("Akka.IO.Udp.Send public constructors are obsolete. Use `Send.Create` or `Send(ByteString, EndPoint, Event)` instead.")]
-            public Send(IEnumerator<ByteBuffer> payload, Event ack)
```
Comment (Arkatufus) [inline src/core/Akka/IO/UdpConnected.cs:169]: Obsolete since v1.3.13 (6 years)

File: src/core/Akka/IO/UdpSettings.cs [hunk 4]
```diff
@@ -46,27 +46,6 @@ public static UdpSettings Create(Config config)
                 fileIoDispatcher: config.GetString("file-io-dispatcher", "akka.actor.default-dispatcher"));
         }
         
-        // directBufferSize and maxDirectBufferPoolSize parameter are byterot, remove this in v1.5
-        [Obsolete("Please the constructor without directBufferSize and maxDirectBufferPoolSize parameter")]
-        public UdpSettings(
```
Comment (Arkatufus) [inline src/core/Akka/IO/UdpSettings.cs:51]: Obsolete since 1.4.28

**Review by Aaronontheweb — APPROVED** [review]
LGTM

**Pull request merged**